src/Language/PureScript/CodeGen/Externs.hs

```diff
@@ -38,11 +38,24 @@ import Language.PureScript.Types
 moduleToPs :: Module -> Environment -> String
 moduleToPs (Module _ _ _ Nothing) _ = error "Module exports were not elaborated in moduleToPs"
 moduleToPs (Module _ moduleName ds (Just exts)) env = intercalate "\n" . execWriter $ do
-  tell [ "module " ++ runModuleName moduleName ++ " where"]
+  let exps = listExports exts
+  tell ["module " ++ runModuleName moduleName ++ (if null exps then "" else " (" ++ exps ++ ")") ++ " where"]
```

Comment:
No it was me, and I meant to point it out and ask you about it, but completely forgot. Your version was:
```
tell [ "module " ++ runModuleName moduleName ++ " (" ++ listExports exts ++ ") where"]
```
Which resulted in the empty parens. However, that caused a parsing error (of the generated extern), so I thought it wasn't intended. So I guess it needs to be fixed on the parsing side?